src/test/scala/com/mozilla/telemetry/streaming/TestAggregator.scala

```diff
@@ -15,6 +15,12 @@ class TestAggregator extends FlatSpec with Matchers{
   val k = TestUtils.scalarValue
   val app = TestUtils.application
 
+  val spark = SparkSession.builder()
```

Comment:
It looks to me like this initialization should either be moved into the new test case (line 113), or the similar initialization logic at line 25 should be removed.

Reply:
Agreed, we don't need it per-test.